modules/exploits/windows/misc/allmedia_1_6_buffer_overflow.rb

```diff
@@ -0,0 +1,89 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+# Author: Hejap Zairy
+# Date: 1.08.2022
+# Exploit Prof
+# Proof and Exploit:
+#image:https://i.imgur.com/yLrRR2t.png
+#video:https://streamable.com/x4i50c
+
+
+
+require 'msf/core'
+
+class Metasploit4 < Msf::Exploit::Remote
+    Rank = ExcellentRanking
```

Comment:
See https://github.com/rapid7/metasploit-framework/wiki/Exploit-Ranking for more info on exploit rankings. As this is a buffer overflow, this can never have an ExcellentRanking ranking, as there is a chance it might fail. As this exploit does not presently do a version check the appropriate ranking would be GoodRanking. If one could do a remote version check I'd bump this up to GreatRanking though.

```suggestion
    Rank = GoodRanking
```